### ISSUE TITLE
CFC Display

### ISSUE THREAD

**Issue by WD5Y** [ramdor/Thetis#373]
Feature request: 
Ability to view CFC display (picCFC) as a form to place on screen.

Thanks,
Joe

**Comment by WD5Y** [comment]
Maybe add a container in the "multi-meter" tab

Thanks,
Joe

**Comment by ramdor** [comment]
Hi Joe, yep I have had plans to add this to the meter containers, perhaps the vac ones, and spectrum display data around the passband, but not finding much coding energy right now to do much. atb and a HNY when it arrives. Richie.

**Comment by WD5Y** [comment]
I understand, I tried to go in and pass the "picCFC PictureBox" over to the console but way over my head at this point, Ha, but learning much in the process. You have definitely put the time in on Thetis lately for sure! Have a Happy New Year Ritchie and thanks.

73's
Joe

**Comment by ramdor** [comment]
yes will implement this into the multimeter system at some point

**Comment by Richz28** [comment]
This really would be great feature to have in a container,
